components/movementsensor/adxl345/adxl345.go

```diff
@@ -41,9 +41,30 @@ var modelName = resource.NewDefaultModel("accel-adxl345")
 
 // AttrConfig is a description of how to find an ADXL345 accelerometer on the robot.
 type AttrConfig struct {
-	BoardName              string `json:"board"`
-	I2cBus                 string `json:"i2c_bus"`
-	UseAlternateI2CAddress bool   `json:"use_alternate_i2c_address,omitempty"`
+	BoardName              string   `json:"board"`
+	I2cBus                 string   `json:"i2c_bus"`
+	UseAlternateI2CAddress bool     `json:"use_alternate_i2c_address,omitempty"`
+	EchoInterrupt          string   `json:"echo_interrupt_pin,omitempty"`
+	InterruptsEnabled      []string `json:"interrupts_enabled,omitempty"`
+	TapX                   bool     `json:"tap_x,omitempty"`
+	TapY                   bool     `json:"tap_y,omitempty"`
+	TapZ                   bool     `json:"tap_z,omitempty"`
+	ThreshTap              byte     `json:"thresh_tap,omitempty"`
+	Dur                    byte     `json:"dur,omitempty"`
+	Latent                 byte     `json:"latent,omitempty"`
+	Window                 byte     `json:"window,omitempty"`
+	ThreshFF               byte     `json:"thresh_ff,omitempty"`
```

Comment:
big +1. let's limit the feature set and simplify this list of features...this won't be very efficient on a pr.
@kim-mishra can you throw these into a doc w/ a description of features (e.g. single tap, double tap, free fall) and what config parameters (e.g. tapx,y,z) will be needed for it?

At the moment i think single tap and free fall is all we need. activity, double tap, and inactivity aren't worth the config noise imo.

Reply:
Sounds good, `double_tap` is not much more of a lift from `single_tap` because all the config registers are the same. Should we keep it or remove for simplicity?

Reply:
Do you have a use case in mind? I'm open to leaving it in if you can think of a robot that cares whether it's been tapped twice, but off the top of my head I can't think of that, so am inclined to get rid of it. We can add it in later if someone actually wants it, but before then our implementation is likely to be kludgy because we don't actually know what anyone wants from this.

Reply:
Sounds good that makes sense to me. @stevebriskin Where is the best place to create this doc?